source/System.cpp

```diff
@@ -966,6 +974,13 @@ int System::MinimumFleetPeriod() const
 
 
 
+int System::LingerTime() const
+{
+	return lingerTime >= 0 ? lingerTime : minimumFleetPeriod;
```

Comment:
It used to be minimumfleetperiod/4, why the change?

Reply:
That's an error. It should be `minimumFleetPeriod / 4`.

Reply:
And now it is.